### ISSUE TITLE
Pretty up the sidebar/header thing

### ISSUE THREAD

**Issue by edrex** [pdxruby/pdxruby.github.io#28]
Maybe create a vertical variant of the PDX.rb logo

**Comment by jc00ke** [comment]
@sproutworx do you still have the original PSD/AI files?

**Comment by Sproutworx** [comment]
Yup. They are on my old machine though. I'll fire it up this weekend and send your way!

**Comment by reidab** [comment]
Looks like I had a copy of those: [pdxruby_logos.zip](https://github.com/pdxruby/pdxruby.github.io/files/334765/pdxruby_logos.zip)

**Comment by edrex** [comment]
I'll do a first pass on this unless someone else wants it.
